projects/projekt-lol/projekt-lol.json

```
@@ -0,0 +1,43 @@
+{
+  "id": "projekt-lol",
+  "slug": "projekt-lol",
+  "name": "projekt.lol",
+  "category": "defi",
+  "chains": [
+    "moonbeam"
+  ],
+  "logo": {
+    "small": {
+      "fileName": "projekt-lol-logo-small.png",
+      "width": 36,
+      "height": 36,
+      "mimeType": "image/png"
+    },
+    "large": {
+      "fileName": "projekt-lol-logo-large.png",
+      "width": 179,
+      "height": 179,
+      "mimeType": "image/png"
+    },
+    "full": {
+      "fileName": "projekt-lol-logo-full.png",
+      "width": 3000,
+      "height": 3000,
+      "mimeType": "image/png"
+    }
+  },
+  "shortDescription": "Launch your own memecoin on Moonbeam with just a few clicks!",
+  "description": "projekt.lol is your go-to platform for all the hottest emerging memecoins on Moonbeam. As a memecoin incubator, the platform enables you to invest in early-stage tokens or create your own memecoin with just a few clicks. The token becomes active as soon as it's deployed on the site, allowing for immediate buying and selling. Memes are instantly tradeable, without requiring any initial liquidity.",
+  "tags": [
+    "DeFi",
+    "GLMR Grants",
```

Please fix the JSON. It's invalid.

```suggestion
    "GLMR Grants"
```